src/ripple/app/tx/impl/NFTokenMint.cpp

```diff
@@ -146,6 +186,54 @@ NFTokenMint::preclaim(PreclaimContext const& ctx)
             return tecNO_PERMISSION;
     }
 
+    if (auto const amount = ctx.tx[~sfAmount])
+    {
+        if (hasExpired(ctx.view, ctx.tx[~sfExpiration]))
+            return tecEXPIRED;
+
+        if (!(ctx.tx.getFlags() & nft::flagCreateTrustLines) &&
```

Comment:
I don't think this check is necessary because the preflight already checked `NFTokenMintMask` to exclude the old `flagCreateTrustLines` flag